docs/pages/understand/understand.md

```diff
@@ -26,7 +26,7 @@ The execution of an automation script can be associated with the following statu
 
 |Status|Description|
 |------|-----------|
-|ACTIVE|Script is running|
+|RUNNING|Script is running|
```

Comment:
killed?

Reply:
KILLED is not a possible script run status. See core/java/iesi-core/src/main/java/io/metadew/iesi/common/configuration/ScriptRunStatus.java for all options. SKIPPED is omitted for now.